streams/src/main/java/org/apache/kafka/streams/kstream/internals/MaterializedInternal.java

```diff
@@ -40,6 +40,14 @@ public void generateStoreNameIfNeeded(final InternalNameProvider nameProvider,
         }
     }
 
+    public boolean isQueryable() {
+        return queriable;
+    }
+
+    public String queryableStoreName() {
```

Comment:
Why do we need this? Can't we call `storeName()` directly? If the code base is clean, `storeName()` should return `null` iff table is not queryable?

Reply:
The issue is that we may need to create internal name via `generateStoreNameIfNeeded` which will set the `storeName` field. Ditto for the question around `isQueryable`.

Reply:
Ack.

Reply:
Ack. I think we can just remove `isQueryable` actually since for whichever its caller, we can always use queryableName instead.